### DISCUSSION TITLE
Nested vertical ScrollViews, take over control once reached a point on Y axis

### DISCUSSION THREAD

**Discussion by krakz999** [software-mansion/react-native-gesture-handler#1186] [Q&A]
## Description

I have a scrollable screen, and inside this screen a scrollable tab view (https://github.com/satya164/react-native-tab-view/). I'm curious if the following is possible with RNGH:

Once the outer scroll view is scrolled to the point where the tab view is on the top, let the inner scrollview(s) take over the gesture, each tab keeping their own scroll state, but whenever the current inner scrollview is scrolled upwards, at y=0 the outer scrollview would take over the gesture.

Without scrolling, we control the parent scrollview here.
![Frame 1](https://user-images.githubusercontent.com/22370549/92409652-a0226b80-f141-11ea-9ec1-3c500c06826d.png)

Reached the tab view, we control the inner scrollview here as long as we don't exceed 0 in the negative direction, at that point we "give" back the gesture to the outer scrollview.
![Frame 2](https://user-images.githubusercontent.com/22370549/92409655-a1ec2f00-f141-11ea-90f1-77947ab8170a.png)

**Comment by AkashBabu** [comment]
Can you check if this library helps you: https://github.com/PedroBern/react-native-collapsible-tab-view
